eth2deposit/intl/en/cli/generate_keys.json

```diff
@@ -0,0 +1,35 @@
+{
+    "validate_password": {
+        "msg_password_prompt": "Type the password that secures your validator keystore(s)",
```

Comment:
Forwarding on a comment by @samajammin on initial intl audit (#180) that I agree with... consider making this clearer that the user is about to create a new password, vs typing an existing password, eg:
`Create a password that will secure your validator keystore(s)`
"Type the password that secures" kinda implies there is already password currently securing it.